src/modules/gatsby-plugin/createImgixGatsbyImageDataFieldConfig.ts

```diff
@@ -89,8 +88,8 @@ const resolveGatsbyImageData = <TSource>({
     )(
       resolveDimensions({
         url,
-        manualHeight: O.fromNullable(manualHeight),
-        manualWidth: O.fromNullable(manualWidth),
+        manualHeight: manualHeight,
+        manualWidth: manualWidth,
```

Comment:
nit: do we need to k:v here? Couldn't this just be
```suggestion
        manualHeight,
        manualWidth,
```

Reply:
Thanks, addressed in a follow-up PR!